### PR TITLE
Ug adapted v1

### LINKED ISSUE #43: Add Quick start info

### LINKED ISSUE #45: Change description

### PR DESCRIPTION
**Pull request by CarrieLi1015**
# Changes
Update user guide.
* Added quick start information.
* Changed description.

# Edit feature
resolves #43 
resolves #45

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
 
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `MATHUTORING.jar` (Stay in tune for our release!).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Copy the file to the folder you want to use as the _home folder_ for your MATHUTORING.
```
Comment (toh-xinyi) [inline docs/UserGuide.md:20]:
```suggestion
3. Copy the file to the folder you want to use as the _home folder_ for you to keep track of your students using MATHUTORING.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar MATHUTORING.jar` command to run the application.<br>
+   <br>
+   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data for your reference, you can also choose to delete the sample data after you get familiar with the MATHUTORING.<br>
```
Comment (toh-xinyi) [inline docs/UserGuide.md:24]: 👍

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,41 +1,43 @@
...
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add n/John p/86232384 a/Ang Mo Kio Ave 10 e/JOHN_08@gmail.com` : Adds a contact named `John` to the Address Book.
```
Comment (toh-xinyi) [inline docs/UserGuide.md:32]:
```suggestion
   * `add n/John p/86232384 a/Ang Mo Kio Ave 10 e/JOHN_08@gmail.com c/98763928` : Adds a student named `John` to the student list in MATHUTORING.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,41 +1,43 @@
...
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar MATHUTORING.jar` command to run the application.<br>
+   <br>
+   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data for your reference, you can also choose to delete the sample data after you get familiar with the MATHUTORING.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * `list` : Lists all contacts.
```
Comment (toh-xinyi) [inline docs/UserGuide.md:30]:
```suggestion
   * `listStudents` : Lists all students in the student list.
```

**Review by toh-xinyi — COMMENTED** [review]
Other than some minor changes to be made, LGTM! 😄

File: docs/UserGuide.md (hunk 4 above)
Comment (CarrieLi1015) [reply docs/UserGuide.md]: Thank you for the comments! The user guide is now updated based on your comments.

**Review by toh-xinyi — APPROVED** [review]
LGTM! 👍

**Pull request merged**